ui/src/message_popup/playtest/danson_experience_page.jsx

```diff
@@ -122,64 +102,107 @@ export default React.createClass({
     return (
       <IntroWithEmail defaultEmail={this.state.email} onDone={this.onStart}>
         <div>
-            <p>Welcome!</p>
-            <p>You will go through a set of scenarios that simulate the conversation between you and Mrs. Danson.</p>
-            <p>You need to provide an audio response to each prompt. Please respond as quickly as possible (like you would do in a real conversation).</p>
-            <p>This may feel uncomfortable at first, but it's better to feel uncomfortable here than with a real parent.</p>
+          <p>Welcome!</p>
+          <p>You will go through a couple of activities that are designed to make you better prepared for a potentially difficult parent-teacher conference you might experience as a new teacher.</p>
+          <p>You need to use a computer/laptop that has a mic because you will need to do audio recordings.</p>
+          <p>This simulation is based on work that has been done by Professor Benjamin Dotger at Syracuse University as documented in his book: "I Had No Idea" Clinical Simulations for Teacher Development.</p>
+          <p>This activity would take about 30minutes.</p>
         </div>
       </IntroWithEmail>
     );
   },
 
   renderQuestionEl(question:QuestionT, onLog, onResponseSubmitted) {
+    if (question.stage === 'scenario') {
+      if (question.choices.length > 0) {
+        return <RecordThenClassifyQuestion
+          key={question.id}
+          question={question}
+          choices={question.choices}
+          onLogMessage={onLog}
+          onResponseSubmitted={onResponseSubmitted}
+          forceResponse={true}
+        />;
+      } else {
+        return (
+          <div>
+          <PlainTextQuestion question={question} />
+            <MinimalOpenResponse
+              responsePrompt=""
+              recordText="Respond"
+              onLogMessage={onLog}
+              forceResponse={true}
+              onResponseSubmitted={onResponseSubmitted}
+            /> 
+          </div>
+        ); 
+      }
+    }
 
-    if (question.choices.length > 0) {
-      return <RecordThenClassifyQuestion
-        key={question.id}
-        question={question}
-        choices={question.choices}
-        onLogMessage={onLog}
-        onResponseSubmitted={onResponseSubmitted}
-        forceResponse={true}
-      />;
-    } else {
+    if (question.stage === 'prereflect') {
       return (
-        <div>
-        <PlainTextQuestion question={question} />
-          <MinimalOpenResponse
-            responsePrompt=""
-            recordText="Respond"
+        <div key={JSON.stringify(question)}>
+          <MixedQuestion question={question} />
+          <MinimalTextResponse
+            forceResponse={true}
+            responsePrompt="Your Answer"
+            recordText="NEXT"
             onLogMessage={onLog}
+            onResponseSubmitted={onResponseSubmitted}
+          />
+        </div>
+      );
+    }
+
+    if (question.stage === 'postreflect') {
```

Comment:
No. The difference is that we show the response summary under each postreflect question.